--- a/src/common/emulatorConfig.h
+++ b/src/common/emulatorConfig.h
@@ -76,7 +76,7 @@
 	bool                   dcc_gpu_clear_enabled       = true;
 	bool                   async_submit_enabled        = true;
 	bool                   gpu_mesh_indirect_enabled   = true;
-	uint32_t               gpu_frames_ahead            = 0;
+	uint32_t               gpu_frames_ahead            = 1;
 	uint32_t               label_flush_interval_us     = 2000;
 	uint32_t               gpu_timestamp_scale_percent = 100;
 	bool                   pipeline_libraries_enabled  = true;
